DownFall/Views/MenuSpriteNode.swift

```diff
@@ -35,44 +39,184 @@ class MenuSpriteNode: SKSpriteNode {
                 return ButtonIdentifier.resume
             case .gameWin:
                 return ButtonIdentifier.playAgain
+            case .rotate:
+                return ButtonIdentifier.rotate
             }
             
         }
+        
+        var widthCoefficient: CGFloat {
+            switch self {
+            case .rotate:
+                return 0.9
+            default:
+                return 0.7
+            }
+        }
+        
+        var heightCoefficient: CGFloat {
+            switch self {
+            case .rotate:
+                return 0.65
+            default:
+                return 0.33
+            }
+        }
     }
     
     init(_ menuType: MenuType, playableRect: CGRect, precedence: Precedence) {
-        let menuSizeWidth = playableRect.size.width * 0.7
-        let menuSizeHeight = playableRect.size.height * 0.33
+        let menuSizeWidth = playableRect.size.width * menuType.widthCoefficient
+        let menuSizeHeight = playableRect.size.height * menuType.heightCoefficient
         
         
-        super.init(texture: SKTexture(imageNamed: "menu"),
-                   color: .black,
+        
+        super.init(texture: nil,
+                   color: .menuPurple,
                    size: CGSize(width: menuSizeWidth, height: menuSizeHeight))
         
-        setupButtons(menuType, playableRect, precedence: precedence)
+        removeAllChildren()
+        
+        let border = SKShapeNode(rect: self.frame)
+        border.strokeColor = UIColor.darkGray
+        border.lineWidth = 10.0
+        addChild(border)
+        
         zPosition = precedence.rawValue
+        setupButtons(menuType, playableRect, precedence: precedence)
+        
+    }
+    
+    private func setupButtons(_ menuType: MenuType, _ playableRect: CGRect, precedence: Precedence) {
+        let menuSizeWidth = playableRect.size.width * menuType.widthCoefficient
+        let menuSizeHeight = playableRect.size.height * menuType.heightCoefficient
+        let buttonSize = CGSize(width: menuSizeWidth * 0.4, height: 120)
+        
+        var addDefaultButton = true
+        
+        if menuType == .rotate {
+            
+            // Heights and widths
+            let topHeight: CGFloat = menuSizeHeight/3 - 56
+            let bottomHeight: CGFloat = -menuSizeHeight/3 + 56
+            
+            let rightWidth: CGFloat = menuSizeWidth/2 - 56
+            let leftWidth: CGFloat = -menuSizeWidth/2 + 56
```

Comment:
All these numbers should be captured in constants.  Add a TODO if you dont feel like doing it now